### ISSUE TITLE
compute_contrast function contains false keyword documentation

### ISSUE THREAD

**Issue by JohannesWiesner** [nistats/nistats#439]
`nistats.contrasts.compute_contrast` contains a keyword argument `regression_result`, but the documentation contains only a description of a keyword called 'results'. I guess that's a typo? 

https://nistats.github.io/modules/generated/nistats.contrasts.compute_contrast.html#nistats.contrasts.compute_contrast

**Comment by bthirion** [comment]
Indeed, it's a typo. Thx for reporting. Can you open a PR to fix it ?

**Comment by JohannesWiesner** [comment]
Yes, I can do that.
